osu.Game.Rulesets.Osu/Skinning/LegacyMainCirclePiece.cs

```diff
@@ -23,18 +23,24 @@ public LegacyMainCirclePiece()
             Size = new Vector2(OsuHitObject.OBJECT_RADIUS * 2);
         }
 
+        private Sprite hitCircleSprite, hitCircleOverlay;
+        private SkinnableSpriteText hitCircleText;
+
         private readonly IBindable<ArmedState> state = new Bindable<ArmedState>();
         private readonly Bindable<Color4> accentColour = new Bindable<Color4>();
         private readonly IBindable<int> indexInCurrentCombo = new Bindable<int>();
 
+        [Resolved]
```

Comment:
why was this moved?

Reply:
because of value changed bindings. I think it's safer to bind value changed events in `LoadComplete`?

Reply:
This isn't used by them.

Reply:
I don't follow, what isn't used by them?

Reply:
The thing this comment is attached to. I'm not sure how to be more clear.

Reply:
oh, you meant the `drawableObject` being resolved with the attribute, I thought you meant why I moved the bindings, sorry, will return back.